src/test/java/com/hotel/mappers/book/request/GuestMapperTest.java

```diff
@@ -0,0 +1,48 @@
+package com.hotel.mappers.book.request;
+
+import com.hotel.service.common.Address;
+import com.hotel.service.common.UserInfo;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.opentravel.ota._2003._05.ArrayOfResGuestsTypeResGuest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(MockitoJUnitRunner.class)
+public class GuestMapperTest {
+    @InjectMocks
+    GuestMapper guestMapper;
+
+    @Test
+    public void testMap() {
+        UserInfo userInfo = UserInfo.newBuilder()
+                .setNamePrefix("Mr")
+                .setFirstName("Abc")
```

Comment:
Can we have a meaningful name here. For all the strings in the test class

Reply:
will change later